api/src/main/resources/workbench.yaml

```diff
@@ -2555,7 +2575,8 @@ definitions:
       FEATURED_WORKSPACE_ADMIN,
       COMMUNICATIONS_ADMIN,
       SECURITY_ADMIN,
-      INSTITUTION_ADMIN
+      INSTITUTION_ADMIN,
+      WORKSPACES_VIEW
```

Comment:
Can you add an entry describing this authority to the comment above?

Re: naming, I'm pretty OK with WORKSPACES_VIEW though it still doesn't feel like we've nailed it. Spitballing some other options, definitely feel free to ignore:

- VIEW_ALL_WORKSPACES
- WORKSPACE_ADMIN_VIEW
- WORKSPACE_VIEW_ALL
- WORKSPACE_ADMIN_VIEWER